### PR TITLE
eslint and prettier added to the framework and files formatted

### PR DESCRIPTION
**Pull request by szucst**
* eslint config added
* prettier config added
* already existing files formatted

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,11 @@
     "test:cypress:response-validation": "cypress run --spec 'cypress/interceptors/response-validation.spec.js'",
     "test:cypress:response-modification": "cypress run --spec 'cypress/interceptors/response-modification.spec.js'",
     "test:cypress:request-validation": "cypress run --spec 'cypress/interceptors/request-validation.spec.js'",
-    "test:cypress:request-modification": "cypress run --spec 'cypress/interceptors/request-modification.spec.js'"
+    "test:cypress:request-modification": "cypress run --spec 'cypress/interceptors/request-modification.spec.js'",
+    "prettier:check": "npx prettier --check .",
```
Comment (mgabor18) [inline package.json]: You can remove `npx` from all scripts, as we migrated to `yarn`, so no additional keywords are necessary.

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "all",
+  "singleQuote": true,
+  "semi": true,
+  "tabWidth": 2,
+  "printWidth": 80
```
Comment (mgabor18) [inline .prettierrc.json]: I would suggest to increase the `printWidth` to around 140-160.

**Review by mgabor18 — CHANGES_REQUESTED** [review]
Please delete `package-lock.json` and use `yarn` instead, to install the necessary packages.

File: .prettierrc.json (hunk 2 above)
Comment (sorosz89) [reply .prettierrc.json]: General wrap lengths are 80, 120, but some use 150. I prefer 120 :D

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": ["google", "prettier"],
```
Comment (sorosz89) [inline .eslintrc.json]: can we extend the eslint:recommended as well? https://eslint.org/docs/rules/

File: .prettierrc.json (hunk 2 above)
Comment (mgabor18) [reply .prettierrc.json]: I got used to 160 with the SDK :D
maybe **`127.6²`** ? (:

Comment (mgabor18) [reply .prettierrc.json:6]: I got used to 160 with the SDK :D
maybe **`127.6²`** ? (:

**Comment by szucst** [comment]
I applied the necessary changes based on the comments.

**Review by sorosz89 — APPROVED** [review]
Looks good! We can add extra rules on demand.

**Review by mgabor18 — APPROVED** [review]

**Pull request merged**